### PR TITLE
Adds a gulag officer's uniform and adds spur boots to the warden's locker.

### PR DESCRIPTION
**Pull request by nesisknesis**
image
image
image

The original sprite had a green camouflage design which was dumb since woodland camouflage in space, and also looked out of place.

New uniform inspired by Revolver Ocelot from MGS3
image

The spurs can be found in the warden's locker and makes a sound when the wearer moves around. 

it's probably annoying.

sprite renamed to GRU_officer from KR_officer

now you can be revolver ocelot (revolvers not included, steal the detectives and a syndicate's revolver)

you're pretty good.

when the warden is a catperson https://www.youtube.com/watch?v=y897tozJeg8

no more security stuff as per suggestion in another pr

#### Changelog

:cl:  Identification
rscadd: Adds spur boots to the warden's locker.  
soundadd: The spurs make a sound when the wearer moves. 
imageadd: Gulag officer uniform added. 
imageadd: Warden's spur jackboots.
/:cl:

### CODE REVIEW

**Comment by drderp3635** [comment]
Make the spurs deal damage to downed people you walk over if you are on harm intent

File: yogstation/code/game/objects/structures/crates_lockers/closets/secure/command.dm [hunk 1]
```diff
@@ -32,7 +32,7 @@
 	..()
 	new /obj/item/clipboard/yog/paperwork/warden(src)
 	new /obj/item/card/id/departmental_budget/sec(src)
-	new /obj/item/clothing/under/yogs/krofficer(src)
```
Comment (fluffe9911) [inline yogstation/code/game/objects/structures/crates_lockers/closets/secure/command.dm:35]: Why are you so intent on replacing outfits with drastically different ones reeee.

**Review by fluffe9911 — CHANGES_REQUESTED** [review]
Stop trying to replace stuff with drastically different stuff.

File: yogstation/code/game/objects/structures/crates_lockers/closets/secure/command.dm (hunk 1 above)
Comment (nesisknesis) [reply yogstation/code/game/objects/structures/crates_lockers/closets/secure/command.dm:35]: duder, woodland camouflage does not belong in space, nor on the security team. i wish to make it contextually sound within spacemans. 
image

Comment (fluffe9911) [reply yogstation/code/game/objects/structures/crates_lockers/closets/secure/command.dm:35]: oh yeah sure but space russians, space wizards, doom esc hell monsters, pretty much all the autodrobe costumes and bloody waldo totally belong in space. Its a gameeeeeee let people dress how they want to dress man.

Comment (nesisknesis) [reply yogstation/code/game/objects/structures/crates_lockers/closets/secure/command.dm:35]: space russians and bloody waldo can do what they want, i'm talking about the security team.

**Comment by Firewolf34** [comment]
NOOOOOOOOOOOO!

I love that green outfit. You gotta leave the green one.

The other one I was cool with replacing because HoS already has so many things but you gotta leave the green one! It isn't green because of woodland camouflage, it's green because of tradition and army olive drab. 

Many armies use olive drab because the color instantly gives the viewer the impression they are dealing with uniformed military, and because of tradition. 

I can understand how in space, in the year 2550, we probably would have moved away from some of these traditions, but the Olive Drab is there because the warden has a bit of a military meme to him. You can't take my olive drab! :(

I like the new uniform, but make it additional to the Olive Drab pleeeaaseee

I'll understand if you end up changing it, because, yes, it does make sense... but I just really really am biased towards that uniform. Hell I even got my entry into the Yogstation Wiki Contest wearing this uniform, so if you remove it, the Yogstation Wiki will be out of date! Consider that, huh? :)
3582-fe95188cbb79ec117268479e17e73055 jpg

**Comment by nesisknesis** [comment]
k

**Review by fluffe9911 — APPROVED** [review]

**Comment by Alexandria404** [comment]
Add it as an optional one?

**Comment by Firewolf34** [comment]
I just wanted to say, thank you @Ihonkification for keeping that beautiful, luscious green jacket in the game :)

And great work on the PR. Love the boots.

**Pull request merged**